ingestion-beam/src/main/java/com/mozilla/telemetry/metrics/KeyedCounter.java

```diff
@@ -11,7 +11,7 @@
  */
 public class KeyedCounter {
 
-  static final Map<String, Counter> counters = new HashMap<>();
+  static final Map<String, Counter> counters = new ConcurrentHashMap<>();
```

Comment:
> LGTM once we understand the failing test here.

Concurrent access made it *appear* that the mapping function modified this map, and there is a change in HashMap.computeIfAbsent method specification for java 11:

> Throws:ConcurrentModificationException - if it is detected that the mapping function modified this map